.github/workflows/build-test.yml

```diff
@@ -110,6 +110,12 @@ jobs:
           distribution: temurin
           java-version: 8
 
+      - name: Install go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '^1.20'
+        run: go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.32.0
+
```

Comment:
Ah looks like we have to split this into two: "a step cannot have both the `uses` and `run` keys"